lib/jxl/enc_external_image.cc

```diff
@@ -268,7 +268,8 @@ Status BufferToImageBundle(const JxlPixelFormat& pixel_format, uint32_t xsize,
   JXL_RETURN_IF_ERROR(ConvertFromExternal(
       jxl::Span<const uint8_t>(static_cast<const uint8_t*>(buffer), size),
       xsize, ysize, c_current,
-      /*alpha_is_premultiplied=*/false, bitdepth, pixel_format, pool, ib));
+      /*alpha_is_premultiplied=*/ib->AlphaIsPremultiplied(), bitdepth,
```

Comment:
we could read out ib->AlphaIsPremultiplied inside the function, since we are passing ib as parameter anyway

Reply:
I tried to remove the argument `alpha_is_premultiplied` from the function `ConvertFromExternal`, but here it seems that it is needed:

https://github.com/libjxl/libjxl/blob/8632a76da0884ca685b71cdb76756bdef4417f60/lib/extras/packed_image_convert.cc#L59-L62

What do you think?

Reply:
I don't think that it is really needed there either, since bundle->AlphaIsPremultiplied() should return the same as info.alpha_premultiplied.

Reply:
Please check that this makes sense...